subsys/net/ip/Kconfig

```diff
@@ -529,6 +529,22 @@ config NET_DEFAULT_IF_CANBUS
 
 endchoice
 
+config NET_IF_USERSPACE_ADDR_ADD
```

Comment:
Why having such atomic options? Couldn't we have a unique generic NET_IF_USERSPACE  instead?
If we expose more to the userspace following the same pattern, we will end up with a long list of Kconfig options.

Reply:
I was thinking a bit more fine grained configuration options, but we can of course go with wider options too. I have no big preference here.